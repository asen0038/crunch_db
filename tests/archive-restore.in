set a 1 2 3 4
set b 4 5 6
snapshot
archive 1 data.txt
set c 3 4 5
snapshot
rollback 2
list entries
restore data.txt
list entries
bye
