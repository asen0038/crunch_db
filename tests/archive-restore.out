>
ok

>
ok

>
saved as snapshot 1

>
ok

>
ok

>
saved as snapshot 2

>
ok

>
c [3 4 5]
b [4 5 6]
a [1 2 3 4]

>
ok

>
b [4 5 6]
a [1 2 3 4]

>
bye
